lib/log.dart

```diff
@@ -30,3 +33,45 @@ bool debugLog(String message) {
   }());
   return true;
 }
+
+/// Display an error message.
+///
+/// This shows a [SnackBar] containing the message after app startup,
+/// otherwise logs it to the console.
+///
+/// See also: [ZulipApp._reportErrorToUserBriefly]
+void Function(String message) get reportErrorToUserBriefly => (message) {
```

Comment:
How about `reportErrorToUserInSnackbar`, to be more informative, and more parallel with `reportErrorToUserInDialog`?

Reply:
Yeah that sounds like a good choice.